blocklylib-core/src/main/java/com/google/blockly/android/ui/WorkspaceView.java

```diff
@@ -59,6 +59,10 @@ public WorkspaceView(Context context, AttributeSet attrs, int defStyleAttr) {
         mTouchSlop = touchSlop;
     }
 
+    public BlocklyController getController() {
```

Comment:
Remove, per comment in `VerticalBlockViewFactory`.

Reply:
See my other comment.